vizro-core/changelog.d/20250214_151624_4648633+gtauzin_kedro_api.md

```diff
@@ -0,0 +1,47 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Highlights ✨
+
+- A bullet item for the Highlights ✨ category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX. ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+<!--
+### Removed
+
+- A bullet item for the Removed category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX. ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+
+### Added
+
+- Kedro integration functions were added to the API reference. ([#1020](https://github.com/mckinsey/vizro/pull/1020))
+
```

Comment:
```suggestion
```

Normally we try and keep our changelog just about package changes rather than docs.